android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt

```diff
@@ -0,0 +1,276 @@
+          colorDrawable.setCornerRadius(borderRadius)
+          if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP
+            && selectable is RippleDrawable) {
+            val mask = PaintDrawable(Color.WHITE)
+            mask.setCornerRadius(borderRadius)
+            selectable.setDrawableByLayerId(android.R.id.mask, mask)
+          }
+        }
+        applyRippleEffectWhenNeeded(selectable)
+        val layerDrawable = LayerDrawable(arrayOf(colorDrawable, selectable))
+        background = layerDrawable
+      }
+    }
+
+    private fun createSelectableDrawable(): Drawable {
+      val version = Build.VERSION.SDK_INT
+      val identifier = if (useBorderlessDrawable && version >= 21) SELECTABLE_ITEM_BACKGROUND_BORDERLESS else SELECTABLE_ITEM_BACKGROUND
+      val attrID = getAttrId(context, identifier)
+      context.theme.resolveAttribute(attrID, resolveOutValue, true)
+      return if (version >= 21) {
+        resources.getDrawable(resolveOutValue.resourceId, context.theme)
+      } else {
+        @Suppress("Deprecation")
+        resources.getDrawable(resolveOutValue.resourceId)
+      }
+    }
+
+    override fun onLayout(changed: Boolean, l: Int, t: Int, r: Int, b: Int) {
+      // No-op
+    }
+
+    override fun drawableHotspotChanged(x: Float, y: Float) {
+      if (responder == null || responder === this) {
+        super.drawableHotspotChanged(x, y)
+      }
+    }
+
+    override fun setPressed(pressed: Boolean) {
+      if (pressed && responder == null) {
+        // first button to be pressed grabs button responder
+        responder = this
+      }
+      if (!pressed || responder === this) {
+        // we set pressed state only for current responder
+        super.setPressed(pressed)
+      }
+      if (!pressed && responder === this) {
+        // if the responder is no longer pressed we release button responder
+        responder = null
+      }
+    }
+
+    override fun dispatchDrawableHotspotChanged(x: Float, y: Float) {
+      // No-op
+      // by default Viewgroup would pass hotspot change events
+    }
+
+    companion object {
+      const val SELECTABLE_ITEM_BACKGROUND = "selectableItemBackground"
+      const val SELECTABLE_ITEM_BACKGROUND_BORDERLESS = "selectableItemBackgroundBorderless"
+
+      var resolveOutValue = TypedValue()
+      var responder: ButtonViewGroup? = null
+      var dummyClickListener = OnClickListener { }
+
+      @TargetApi(Build.VERSION_CODES.LOLLIPOP)
+      private fun getAttrId(context: Context, attr: String): Int {
+        SoftAssertions.assertNotNull(attr)
+        return when {
+          SELECTABLE_ITEM_BACKGROUND == attr -> {
+            R.attr.selectableItemBackground
+          }
+          SELECTABLE_ITEM_BACKGROUND_BORDERLESS == attr -> {
+            R.attr.selectableItemBackgroundBorderless
+          }
+          else -> {
+            context.resources.getIdentifier(attr, "attr", "android")
+          }
+        }
```

Comment:
We can move `attr` directly to `when` clause
```suggestion
        return when (attr) {
          SELECTABLE_ITEM_BACKGROUND -> {
            R.attr.selectableItemBackground
          }
          SELECTABLE_ITEM_BACKGROUND_BORDERLESS -> {
            R.attr.selectableItemBackgroundBorderless
          }
          else -> {
            context.resources.getIdentifier(attr, "attr", "android")
          }
        }
```

Reply:
Fixed in e37404e.